doc/USERMANAGEMENT.md

```diff
@@ -173,20 +171,181 @@ Now you can go to `import`, paste that template and click `proceed` and the user
+
+We are going to create a group called **devs** and add a user in that group and then **configure each service** to authenticate that particular group. The LDAP protocol do not define how programs function either on the server or client, but the messages exchanged between an LDAP server and an LDAP client, to manage your users well you need to know how to create users/groups in the LDAP server and then you need to assign every user/group to the right service or application **through the application's configuration in the `values.yaml` file**.
+
+We are going to create a group called **devs** and a group called **admins** and add a user in each group and then **configure each service** to authenticate the newly created users/groups.
+
+#### Create groups in openldap
+
+When using openldap you will probably need to create different groups and give them different roles or assign them on different services. To manage your LDAP server, head to phpLDAPadmin:
+
+here's a simple ldif code to import that will create:
+
+* An Organizational Unit `OU=people`
+* A group called **admins** under `ou=people,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=admins,ou=people,dc=ldap,dc=cetic,dc=be`
+* A user called `John` under `cn=admin,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=john,cn=admins,ou=people,dc=ldap,dc=cetic,dc=be` with the password `john123`
+* A group called **devs** under `ou=people,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=devs,ou=people,dc=ldap,dc=cetic,dc=be` 
+* A user called `Luke` under `cn=devs,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=luke,cn=devs,ou=people,dc=ldap,dc=cetic,dc=be` with the password `luke123`
+
+```
+dn: ou=people,dc=ldap,dc=cetic,dc=be
+ou: people
+objectClass: organizationalUnit
+
+dn: cn=admins,ou=people,dc=ldap,dc=cetic,dc=be
+cn: admins
+gidnumber: 501
+objectclass: posixGroup
+objectclass: top
+
+dn: cn=devs,ou=people,dc=ldap,dc=cetic,dc=be
+cn: devs
+gidnumber: 500
+objectclass: posixGroup
+objectclass: top
+
+dn: cn=luke,cn=devs,ou=people,dc=ldap,dc=cetic,dc=be
+cn: luke
+gidnumber: 500
+givenname: luke
+homedirectory: /home/users/lskywalker
+loginshell: /bin/sh
+objectclass: inetOrgPerson
+objectclass: posixAccount
+objectclass: top
+sn: skywalker
+uid: luke
+uidnumber: 1000
+userpassword: {MD5}hSQr2UGesHOpB9f3VrX43Q==
+
+dn: cn=john,cn=admins,ou=people,dc=ldap,dc=cetic,dc=be
+cn: john
+gidnumber: 501
+givenname: john
+homedirectory: /home/users/John
+loginshell: /bin/sh
+objectclass: inetOrgPerson
+objectclass: posixAccount
+objectclass: top
+sn: Doe
+uid: john
+uidnumber: 1001
+userpassword: john123
+```
 
+## PostgreSQL
```

Comment:
This title and the following should be numbered and included in the TOC of the page.